step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManager.java

```diff
@@ -355,23 +355,27 @@ protected List<ExecutiontTaskParameters> prepareExecutionTasksParamsStaging(Obje
             execTaskParameters.setActive(schedule.getActive() == null || schedule.getActive());
             execTaskParameters.addAttribute(AbstractOrganizableObject.NAME, schedule.getName());
             execTaskParameters.setCronExpression(schedule.getCron());
+            if (schedule.getAssertionPlanName() != null && !schedule.getAssertionPlanName().isEmpty()) {
```

Comment:
Store schedule.getAssertionPlanName() in a local variable to avoid redundant access

Reply:
Fixed